lib/src/controls/navigation/tab_view.dart

```diff
@@ -774,6 +777,15 @@ class Tab with Diagnosticable {
   /// The body of the view attached to this tab
   final Widget body;
 
+  /// The color of the tab.
+  final Color? color;
+
+  /// The color of the selected tab.
+  final Color? selectedColor;
+
+  /// The  outline color of the tab.
+  final Color? outlineColor;
```

Comment:
These, ideally, would have to be of type `ButtonState<Color>?`. The colors are not static and depend on the current state, specially on a desktop environment.

Reply:
I left these properties with the Color type because they are not part of the documentation, but can be very useful for specific user customizations. For example, provide a static tab color depending on the type of content.